ATTRIBUTION.md

```diff
@@ -447,4 +447,36 @@ The data are used to populate the power generation profile of U.S. transmission
 ##### Note
 Public use (see https://hifld-geoplatform.opendata.arcgis.com/datasets/geoplatform::power-plants/about). Users are advised to read the data set's metadata thoroughly to understand appropriate use and data limitations.
 
+
+##### Source
+* Name: U.S. Zips
+* Author: simplemaps.com
```

Comment:
Not sure whether we should put down the company name in the Author field: `Pareto Software, LLC`?

Reply:
Good call, done.